### ISSUE TITLE
Symptoms improve with nitroglycerin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10131]
**Preferred term label:**
Symptoms improve with nitroglycerin

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
myocardial infarction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000045
- label: Ameliorated by nitroglycerin
- parents: [Ameliorated by exposure to medication (HP:0034536)](https://hpo.jax.org/app/browse/term/HP:0034536)
- pmid: 
- def: Applies to a sign or symptom that is improved or made more bearable by treatment with nitroglycerin.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10131
- synonyms:
